src/main/java/cn/nukkit/ChatColor.java

```diff
@@ -0,0 +1,236 @@
+package cn.nukkit;
+
+import cn.nukkit.exception.ChatColorCharOutOfBoundsException;
+import com.google.common.collect.Maps;
+import com.sun.istack.internal.NotNull;
+
+import java.util.Map;
+
+public enum ChatColor {
+    /**
+     * Represents black
+     */
+    BLACK('0', 0x00),
+    /**
+     * Represents dark blue
+     */
+    DARK_BLUE('1', 0x1),
```

Comment:
Nukkit also have a class which does exactly the same. It is the [`TextFormat`](https://github.com/GameModsBR/PowerNukkit/blob/master/src/main/java/cn/nukkit/utils/TextFormat.java) enum.